Alpaca.Markets/NatsClient.cs

```diff
@@ -8,7 +8,7 @@
 namespace Alpaca.Markets
 {
     /// <summary>
-    /// Provides unified type-safe access for Polygon streaming API.
+    /// Provides unified type-safe access for Polygon streaming API via NATS.
     /// </summary>
     public sealed partial class NatsClient : IDisposable
```

Comment:
Maybe mark this class `Obsolete` attribute with appropriate description and link to some web page explaining upcoming switch?